### PR TITLE
Saleor 1743 tests for variants

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I created tests for variants

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.
1. [ ] The changes are tested in different browsers (Chrome, Firefox, Safari).
1. [ ] The changes are tested in light and dark mode.

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1743 Tests for Catalog -> Products -> Variants](https://app.clickup.com/t/2549495/SALEOR-1743)

File: cypress/integration/products/productsVariants.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+import Channels from "../../apiRequests/Channels";
+import Product from "../../apiRequests/Product";
+import VariantsSteps from "../../steps/products/VariantsSteps";
+import { urlList } from "../../url/urlList";
+import ChannelsUtils from "../../utils/channelsUtils";
+import ProductsUtils from "../../utils/productsUtils";
+import ShippingUtils from "../../utils/shippingUtils";
+import { getProductVariants } from "../../utils/storeFront/storeFrontProductUtils";
+
+// <reference types="cypress" />
```
Comment (mmarkusik) [inline cypress/integration/products/productsVariants.js:12]: remove comment?

File: cypress/integration/products/productsVariants.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        cy.visit(`${urlList.products}${createdProduct.id}`);
+        variantsSteps.createVariant({
+          sku: secondVariantSku,
+          warehouseName: warehouse.name,
+          attributeName: variants[1].name,
+          price: variants[1].amount
+        });
+      })
+      .then(() => getProductVariants(createdProduct.id, defaultChannel.slug))
+      .then(variantsResp => {
```
Comment (mmarkusik) [inline cypress/integration/products/productsVariants.js:123]: We could destructure the response like
```suggestion
      .then(([namedProp, otherNamedProp, anotherNamedProp]) => {
```

File: cypress/steps/products/VariantsSteps.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import { PRODUCTS_SELECTORS } from "../../elements/catalog/product-selectors";
+import { VARIANTS_SELECTORS } from "../../elements/catalog/variants-selectors";
+class VariantsSteps {
```
Comment (mmarkusik) [inline cypress/steps/products/VariantsSteps.js:3]: Blank lines between elements would look nice

File: cypress/integration/products/productsVariants.js (hunk 1 above)
Comment (krzysztofwolski) [reply cypress/integration/products/productsVariants.js:12]: https://docs.cypress.io/guides/tooling/intelligent-code-completion.html#Triple-slash-directives

**Review by krzysztofwolski — APPROVED** [review]

**Review by mmarkusik — APPROVED** [review]

**Pull request merged**